Dockerfile

```diff
@@ -1,6 +1,7 @@
 from ubuntu:18.04
 
 # Install prerequisites
+env DEBIAN_FRONTEND=noninteractive
 run apt-get update && apt-get install -y \
```

Comment:
Don't use `env` for that, use `run DEBIAN_FRONTEND=noninteractive apt-get update...`

See https://github.com/moby/moby/issues/4032

Reply:
Oh, I didn't think about that, thanks! Fixed in https://github.com/openalpr/openalpr/pull/690/commits/61e98b4c8b7010a9f5e5ab82d18e7222755cbd84